apps/api/src/domain/models/resource.entity.ts

```diff
@@ -8,7 +8,7 @@ import { ResourceCompositeIdentifier } from './types/entityCompositeIdentifier';
 export abstract class Resource extends BaseDomainModel {
     abstract readonly type: ResourceType;
 
-    readonly id: ResourceId;
+    readonly id: EntityId;
```

Comment:
I put this back after realizing that our `Edge Connections` are not resources, but have the same format for their ID. Remember this is just an alias for `string` behind the scenes.

Reply:
Be good to talk this through @aaron-plahn to make sure I'm clear.

Reply:
We talked about this.